runelite-client/src/main/java/net/runelite/client/plugins/runecraft/RunecraftOverlay.java

```diff
@@ -53,6 +58,8 @@
 
 	private final RunecraftConfig config;
 
+	private final Map<String, BufferedImage> storedOutlines = new HashMap<>();
```

Comment:
As this is second plugin using outline API and basically duplicating the logic, maybe extending sprite manager with it would make sense